docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md

```diff
@@ -1,48 +1,95 @@
 
-:::note
-String resources in SDK are grouped in the resource files, usually prefixed with `strings_`. Each file corresponds to a related UI component or specific usage. You can browse them in order to provide translations:
-* [strings_attachment_gallery.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_attachment_gallery.xml) 
-* [strings_channel_list.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_channel_list.xml) 
-* [strings_channel_list_header.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_channel_list_header.xml) 
-* [strings_mention_list.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_mention_list.xml) 
-* [strings_message_input.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_message_input.xml)
-* [strings_message_list.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_message_list.xml) 
-* [strings_message_list_header.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_message_list_header.xml) 
-* [strings_search.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_search.xml) 
-* [strings_common.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_common.xml)
-:::
+### Overriding existing languages
+
+To override strings for a language supported by default, create new string resources in the locale-specific `/res/values-XX` directories of your project with the same `name`. [Here](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-components/src/main/res/values) you can find a list of available text resources grouped by component.
+
+### Overriding the default language
+
+To change the strings used by default, place the string resources you want to override inside the `/res/values` [directory](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-components/src/main/res/values) directory.
+
+### Removing existing languages
+
+The Android UI Components include resources for all of the languages mentioned above. If your app doesn't support all those languages, you can can exclude some of them by explicitly defining a list of supported languages inside your `build.gradle` file:
+
+```groovy
+defaultConfig {
+    resConfigs "en", "es"
+}
+```
+
+With the configuration here, your app will include only _English_ and _Spanish_ resources.
```

Comment:
After this, add the following

### Translating User Messages

Stream Chat provide the ability to run users' messages through automatic translation.
While machine translation is never perfect it can enable two users to communicate with
each other without speaking the same language. For more information, see the full guide
to [adding automatic translation](https://getstream.io/chat/docs/android/translation/).